src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -2243,6 +2243,10 @@ class PvtModelTableParametersDescription:
     number_of_phases: int = attr.ib(default=2, validator=instance_of(int))
     warn_when_outside: bool = attr.ib(default=True, validator=instance_of(bool))
 
+    table_type: Optional[str] = attr.ib(
+        default="PT", validator=optional(instance_of(str))
```

Comment:
We should be using an `Enum` here. Take a look in [here](https://github.com/ESSS/alfasim-sdk/blob/master/src/alfasim_sdk/_internal/constants.py) in order to see how we use it. Using strings for representing `_type` can cause problems.

Reply:
Also, can `table_type` be `None`? The `Optional[str]` assumes it can.

Reply:
I created a new `Enum` called `PVTTableType`. Their values are the same as defined in `core_constants` to use the `alfasim-sdk` definition to build the `PvtModel` in the `alfasim_core`.

Under the `PvtModelTableParametersDescription` I defined the `table_type` as an `attrib_enum` referencing the created `Enum`.

Now I am using this `Enum` in `ALFAsim` repo  to create the `PvtModel` in `pvt_model.py`